index.html

```diff
@@ -6,11 +6,91 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Sofia">
     <!-- Add a link to your css file here -->
 </head>
 
 <body>
     <!-- Add your markup here -->
+    <header>
+        <p class="call-to-action-p">Sign up and get <strong>£5 OFF*</strong> your first order <button class="call-to-action-btn">SIGN UP</button></p>    
+        <div class="header-container">
+            <img alt="" class="header-img" src="https://cdn.pixabay.com/photo/2014/04/03/10/07/cupcake-309889__340.png" width="100%" height="100%">
+            
+            <div class="nav-bar">
+            
+            
+                <div class="hamburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+               
+                <p class="nav-bar-p">Lee-Lee Bakes</p>
```

Comment:
Its good practice to wrap the logo/site name in an anchor that links to the homepage of the site

Reply:
Noted, thank you.